src/funding/paypal/template.jsx

```diff
@@ -164,16 +164,16 @@ export function Label(opts : LabelOptions) : ChildType {
 }
 
 export function AnimatedExperimentLabel(opts : AnimatedExperimentLabelOptions) : ChildType {
-    const { buttonAnimationComponent } = opts;
-    delete opts.buttonAnimationComponent;
+    const { buttonDesignComponent } = opts;
```

Comment:
Could we remove the `delete` operation and replace using `...rest`?
```suggestion
    const { buttonDesignComponent, ...updatedOpts } = opts;
```

Reply:
Yes we can do this